### PR TITLE
Add tryton server integration documentation

### PR DESCRIPTION
**Pull request by n1ngu**
Related to https://github.com/getsentry/sentry-python/pull/548#issuecomment-558681302 which is being supervised by @untitaker

### CODE REVIEW

**Comment by n1ngu** [comment]
I'm unfamiliar to Jekyll projects and I still haven't checked the documentation locally. This is barely a copy&paste from the Bottle integration.

**Comment by untitaker** [comment]
idk why but for some reason I can never push to your PRs :)

```diff
3c9b6afe Markus Unterwaditzer 26 seconds ago (HEAD -> trytond)
        fix: Update snapshot

diff --git a/__tests__/__snapshots__/documentation.js.snap b/__tests__/__snapshots__/documentation.js.snap
index 79820e93..060addb9 100644
--- a/__tests__/__snapshots__/documentation.js.snap
+++ b/__tests__/__snapshots__/documentation.js.snap
@@ -56,6 +56,7 @@ Array [
   "_platforms/python/rq.json",
   "_platforms/python/sanic.json",
   "_platforms/python/tornado.json",
+  "_platforms/python/tryton.json",
   "_platforms/python/wsgi.json",
   "_platforms/react-native.json",
   "_platforms/ruby.json",
@@ -364,6 +365,7 @@ Array [
   "platforms/python/serverless/index.html",
   "platforms/python/sqlalchemy/index.html",
   "platforms/python/tornado/index.html",
+  "platforms/python/tryton/index.html",
   "platforms/python/wsgi/index.html",
   "platforms/react-native/cocoapods/index.html",
   "platforms/react-native/codepush/index.html",
```

File: src/collections/_documentation/platforms/python/tryton.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Tryton
+sidebar_order: 4
+---
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/tryton.md]: Could you add `{% version_added 0.13.2 %}` here?

**Comment by n1ngu** [comment]
Probably I can neither push to this PR because I am such a mess with users :upside_down_face: 

I'll have this PR fixed ASAP!

**Comment by untitaker** [comment]
sry @n1ngu can you do 0.14.0 as version instead... some time passed inbetween

**Comment by untitaker** [comment]
0.14.0 is scheduled for release

**Comment by n1ngu** [comment]
I am clueless about the failing pipeline. What's this about? How can I solve it?

**Comment by untitaker** [comment]
No worries, will merge and deploy this as soon as 0.14 lands on pypi

**Comment by untitaker** [comment]
Thanks! 0.14 is out

**Pull request merged**